.prettierignore

```diff
@@ -5,10 +5,20 @@
 # We decided in the team to not format scss files
 *.scss
 
+src/static
+*.svg
+*.jpg
+*.pdf
+*.png
+*.rtf
+.gitignore
```

Comment:
Why should you exclude the gitignore from prettier? I'm against that.

Reply:
Prettier dont know this filetype